tests/unit_tests/test_unit.py

```
@@ -1387,9 +1389,9 @@ def test_isoleucine_match(tiny_config):
     )
     db_model = DbSpec2Pep(tokenizer=tokenizer)
     peptides = [["PEPTLDEK"], ["PEPTIDEK"]]
-    mock_get_candidates = lambda _, precursor_charge: pd.Series(
-        peptides[precursor_charge - 1]
-    )
+
+    def mock_get_candidates(_, precursor_charge):
```

Ruff (0.15.6)

[warning] Missing return type annotation for private function `mock_get_candidates`

(ANN202)